### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Homosanians**
Make README a bit fancier. This is an important part of the page that the user glances at first when going to the GitHub page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,26 @@
-# Medusa-plugin-filestorage-local
+# Local filestorage Medusa plugin
 
-Store your images locally on your installation of medusa e-commerce.
+This repo provides the plugin to get you started serving images using your filesystem. Follow the steps below to get ready. Store your images locally on your installation of [Medusa](https://github.com/medusajs/medusa) e-commerce.
 
-## Available options (default configuration)
+## Adding plugin to Medusa.js Core
 
+1. `yarn add https://github.com/Blechlawine/medusa-plugin-filestorage-local.git`
```
Comment (Blechlawine) [inline README.md]: this should not be the github url, but the npm name `medusa-plugin-filestorage-local`. Also we should not just show the yarn command, but also npm and pnpm

**Review by Blechlawine — CHANGES_REQUESTED** [review]
Looks good, except some small changes

File: README.md (hunk 1 above)
Comment (Homosanians) [reply README.md]: Done

**Review by Blechlawine — APPROVED** [review]
lgtm

**Pull request merged**